docs/modules/sql/pages/mapping-to-maps.adoc

```diff
@@ -41,6 +41,42 @@ OPTIONS (
 )
 ```
 
+If key or value is a Portable or Compact complex type, you must enumerate all required columns 
```

Comment:
> you must enumerate all required columns

Not true for Java, did not test Compact but also should work.

For Portable (and maybe for Compact?) explicit enumeration is only required when the schema of the Portable (maybe also Compact?) object cannot be determined, eg. because it has not yet been registered.

Reply:
And, as discussed today, for Portable it may be insufficient to only register PortableFactory and this only does not register `ClassDefinitions`